### PR TITLE
Add downstream documentation links

### PR DESCRIPTION
**Pull request by lzap**
SSIA

Let me know what I missed

### CODE REVIEW

**Comment by lzap** [comment]
@maximiliankolb

**Comment by maximiliankolb** [comment]
Thank. I will review this on Wednesday.

**Comment by lzap** [comment]
Thanks, I am making these statements up so more accurate statement would be great.

@melcorr the same for Satellite in our case :-)

**Comment by melcorr** [comment]
> Thanks, I am making these statements up so more accurate statement would be great.
> 
> @melcorr the same for Satellite in our case :-)

@ekohl makes a valid point here: https://community.theforeman.org/t/foreman-performance-tuning/23104/7 about the relevance of this. 

I would approve if there was a heavy caveat. I am not interested in maintaining this doc. I tried for three years. It was hell. I can't really cope with the thoughts of this tuning guide being another thing added to my TODO list. I already am so behind.

File: web/content/_index.md [hunk 1]
```diff
@@ -6,7 +6,13 @@ This is work-in-progress documentation site for <a href="https://www.theforeman.
...
+Available upstream documentation sets are:
 
 * [Nightly](/nightly.html)
 * [2.3](/2.3.html)
+
+Other relevant documentation:
+
+* [Red Hat Satellite](https://access.redhat.com/documentation/en-us/red_hat_satellite) is a distribution of Foreman and several plugins including Katello, Remote Execution, Discovery, Bootdisk, OpenSCAP for the enterprise.
+* [ATIX orcharhino](https://docs.orcharhino.com/or/docs/index.html) is a distribution of Foreman and many plugins providing vendor-independent management of Red Hat, Suse, Oracle and Debian/Ubuntu nodes
+* [Red Hat Satellite Performance Tuning](https://access.redhat.com/solutions/1257143) is a set of performance-related resources, a free Red Hat portal account is required to access the content.
```
Comment (melcorr) [inline web/content/_index.md:18]:
```suggestion
* [Red Hat Satellite Performance Tuning](https://access.redhat.com/solutions/1257143) is a set of performance-related resources, a free Red Hat portal account is required to access the content. Note that while some of the guidance here might help you, Red Hat Satellite is a combination of Foreman and Katello. The latest Satellite release is always two versions behind the latest Foreman release
```

I would like some caveat like this added.

**Review by melcorr — CHANGES_REQUESTED** [review]

File: web/content/_index.md (hunk 1 above)
Comment (ekohl) [reply web/content/_index.md:18]: :+1: for a caveat. However, currently 6.8 is based on 2.1 and 2.4 is stable so it's even 3 releases. Perhaps you can phrase it in a way that's less committed to the exact number of versions?

Comment (melcorr) [reply web/content/_index.md:18]: A teetered between "several" and "a few". I thought "a few" was less dramatic. Updated ^

**Comment by lzap** [comment]
Great, merged Mel's update. Waiting for @maximiliankolb from ATIX.

Did I miss other resources? There must be many relevant ones.

File: web/content/_index.md [hunk 2]
```diff
@@ -6,7 +6,13 @@ This is work-in-progress documentation site for <a href="https://www.theforeman.
...
-Available documentation sets are:
+Available upstream documentation sets are:
 
 * [Nightly](/nightly.html)
 * [2.3](/2.3.html)
+
+Other relevant documentation:
+
+* [Red Hat Satellite](https://access.redhat.com/documentation/en-us/red_hat_satellite) is a distribution of Foreman and several plugins including Katello, Remote Execution, Discovery, Bootdisk, OpenSCAP for the enterprise.
+* [ATIX orcharhino](https://docs.orcharhino.com/or/docs/index.html) is a distribution of Foreman and many plugins providing vendor-independent management of Red Hat, Suse, Oracle and Debian/Ubuntu nodes
```
Comment (maximiliankolb) [inline web/content/_index.md:17]:
```suggestion
* [orcharhino by ATIX](https://docs.orcharhino.com/or/docs/index.html) is a distribution of Foreman and many plugins providing vendor-independent management of CentOS, Debian, Oracle Linux, Red Hat Enterprise Linux, SUSE Linux Enterprise Server, and Ubuntu hosts. orcharhino supports Amazon EC2, Google GCE, libvirt, Microsoft Azure, Oracle Linux Virtualization Manager, oVirt, Proxmox, RHV, and VMware vSphere as compute resource providers and Ansible, Puppet, and Salt as configuration management solutions.
```.

File: web/content/_index.md [hunk 3]
```diff
@@ -6,7 +6,13 @@ This is work-in-progress documentation site for <a href="https://www.theforeman.
...
+* [Red Hat Satellite Performance Tuning](https://access.redhat.com/solutions/1257143) is a set of performance-related resources, a free Red Hat portal account is required to access the content. Note that while some of the guidance here might help you, Red Hat Satellite is a combination of Foreman and Katello. The latest Satellite release is always a few versions behind the latest Foreman release
```
Comment (maximiliankolb) [inline web/content/_index.md:18]:
```suggestion
* [Red Hat Satellite Performance Tuning](https://access.redhat.com/solutions/1257143) is a set of performance-related resources, a free Red Hat portal account is required to access the content. Note that while some of the guidance here might help you, Red Hat Satellite is a combination of Foreman and Katello. 

The latest Satellite and orcharhino releases are always behind the latest Foreman release.
The downstream documentation is meant as an extension, not a substitution for first party Foreman documentation.
```

**Comment by maximiliankolb** [comment]
I like the idea of adding links to downstream docs!

What do you think about a somewhat generic warning, mostly regarding versions, but potentially also regarding supported OS, plugins, etc.?

**Comment by melcorr** [comment]
> I like the idea of adding links to downstream docs!
> 
> What do you think about a somewhat generic warning, mostly regarding versions, but potentially also regarding supported OS, plugins, etc.?

On average, how many versions is orcharhino behind Foreman/Katello? We do get queries about why x exists in Foreman and not Satellite. I was debating between "several" and "a few" for a while yesterday. With Satellite, it's 2/3 depending on release dates.

**Comment by maximiliankolb** [comment]
This is difficult to quantify, but we'd be happy with `several version behind Foreman/Katello` (for both Satellite and orcharhino).

File: web/content/_index.md (hunk 3 above)
Comment (lzap) [reply web/content/_index.md:18]: I think that the motivation of the last sentence was to tell the reader that Performance Tuning might be out of date.

You are changing this to a more generic statement about the obvious lag of downstream and upstream. I am not sure if that's needed? @melcorr

**Comment by lzap** [comment]
Thans I merged the ATIX update, I am hesitant with the info about downstream being "older". I am not sure how much useful is that.

**Comment by melcorr** [comment]
> Thans I merged the ATIX update, I am hesitant with the info about downstream being "older". I am not sure how much useful is that.

What if there's been some major change, like db or passenger or Pulp 3, something like that between releases? Pulp 2 content, that kind of thing. We point to yesterday's news. I think a note to mention this is worthwhile. People might just think we skin Foreman and sell it, while Satellite and Foreman have disparities. People might not know.

File: web/content/_index.md (hunk 3 above)
Comment (maximiliankolb) [reply web/content/_index.md:18]: For me, this makes the most sense. My logic:
orcharhino & Satellite != Foreman nightly (same for the Performance Tuning Guide) > a generic warning would be neat.

**Comment by lzap** [comment]
Ok then, I amended the suggested change as well. I am changing the URL of the performance guide now to https://redhatsatellite.github.io/satellite-performance-tuning/ because it was published recently. I also keep the official link as well.

**Review by melcorr — APPROVED** [review]

**Review by maximiliankolb — APPROVED** [review]

**Pull request merged**